example.py

```diff
@@ -118,6 +119,29 @@ def render(self, props):
         return DOM.create_element('a', link_props, href)
 
 
+def BlockFallback(props):
+    type_ = props['block']['type']
+
+    if type_ == 'example-discard':
+        logging.warn('Missing config for "%s". Discarding block, keeping content.' % type_)
+        # Directly return the block's children to keep its content.
+        return props['children']
+    elif type_ == 'example-delete':
+        logging.error('Missing config for "%s". Deleting block.' % type_)
+        # Return None to not render anything, removing the whole block.
+        return None
+    else:
+        logging.warn('Missing config for "%s". Using div instead.' % type_)
+        # Provide a fallback.
+        return DOM.create_element('div', {}, props['children'])
```

Comment:
Quite a basic example but it demonstrates how you would define fallbacks depending on different scenarios, and also potentially do logging.